docs/glossary/README.md

```diff
@@ -0,0 +1,287 @@
+
+## Repository
+
+A repository is defined as a document management system that allows users to manage files and folders via different version control system (SVN or Git).
+
+## Roadmap
+
+In OpenProject, a roadmap is defined as an overview page displaying the versions sorted alphabetically and the work packages assigned to them. The roadmap is displayed in the project navigation when the work package module is activated and a version has been created (project settings).
+
+## Role
+
+In OpenProject, a role is defined as a set of permissions defined by a unique name. Project members are assigned to a project by specifying a user’s, group’s or placeholder user’s name and the role(s) they should assume in the project.
+
+## Single user
+
+## Story points
+
+## Team planner
+
+## Time tracking
+
+Time tracking in OpenProject is defined as a module which allows users to log time on work packages. Once the time tracking module is activated, time can be logged via the action menu of a work package.
+
+## User
+
+In OpenProject, a user is defined as a person who uses OpenProject, described by an identifier. New users can be created in the admin settings. Users can become project members by assigning them a role and adding them via the project settings.
+
+## Widget
+
+## Wiki
+
+In OpenProject, a wiki is defined as a module that allows to use textile-based wiki pages. In order to use the wiki module, it has to be activated in the project settings.
+
+## Workflow
+
+A workflow in OpenProject is defined as the allowed transitions between status for a role and a type, i.e. which status changes can a certain role implement depending on the work package type. Workflows can be defined in the admin settings. For example, you might only developers to be able to set the status "developed". [Read more about work package workflows in OpenProject](https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-workflows/#manage-work-package-workflows)
+
+![glossary-openproject-work package](glossary-openproject-work package.png)
+
+## Work package
+
+In OpenProject, a [work package](https://www.openproject.org/docs/user-guide/work-packages/#overview) is defined as an item in a project. It is a subset of a project that can be assigned to users for execution, such as Tasks, Bugs, User Stories, Milestones, and more. Work packages have a type, an ID and a subject and may have additional attributes, such as assignee, responsible, story points or target version. Work packages are displayed in a project timeline (unless they are filtered out in the timeline configuration) - either as a milestone or as a phase. In order to use the work packages, the work package module has to be activated in the project settings.
+
+![glossary-openproject-sys-admin-edit-workflow](glossary-openproject-sys-admin-edit-workflow.png)
```

Comment:
⚠️ The image for workflow  is linked in section "Work package"

Please provide a human readable image description. (e.g. this is read aloud by screenreaders)
e.g.
`![A workflow in OpenProject](glossary-openproject-sys-admin-edit-workflow.png)
`